modules/device/main.tf

```diff
@@ -381,24 +382,38 @@ resource "aws_ecs_task_definition" "device_task_definition" {
        "privileged": false,
        "name": "${local.device_app_name}",
        "environment": [
-         {
-           "name": "ENVIRONMENT",
-           "value": "${terraform.workspace}"
-         },
-         {
-           "name": "APP_NAME",
-           "value": "${local.device_app_name}"
-         }
+         ${local.ecs_shared_env_vars}
        ],
+       "secrets": [
+          ${local.ecs_shared_ssm_secrets},
+          ${local.ecs_daw_shared_ssm_secrets},
```

Comment:
Acronyms for Device, API, WWW and Device, API

Had parameters that were not shared amongst all so I did this...I considered making a variables module and setting the ones used, but felt like that would be more revamping than would be accepted on a public repo.

Reply:
Oh gotcha. Yeah this seems fine. I know it might make it a long variable name but maybe we add the full names like `local.ecs_device_api_www_shared_ssm_secrets` or something similar. 🤔

Reply:
Yeah that's fine by me - should be updated ✅